geopy/geocoders/osm.py

```diff
@@ -248,6 +249,58 @@ def reverse(
             self._call_geocoder(url, timeout=timeout), exactly_one
         )
 
+    def lookup(
+            self,
```

Comment:
pep8/flake8 allows different options, and you decide which of those looks better in each case.

For me, I would have put it into a single line.